crypto-volatility-index/src/derivativesDataProvider.js

```diff
@@ -0,0 +1,119 @@
+const { AdapterError, Requester, logger } = require('@chainlink/external-adapter')
+const axios = require('axios')
+const moment = require('moment')
+
+const EXCHANGE_URL = `https://www.deribit.com/api/v2/public`
+const currencyEndpoint = `${EXCHANGE_URL}/get_index`
+const bookDataEndpoint = `${EXCHANGE_URL}/get_book_summary_by_currency`
+
+const DATE_FORMAT = 'DDMMYYYY'
+
+const getDerivativesData = async (cryptoCurrencies) => {
+  const currencyValues = await Promise.all(
+    cryptoCurrencies.map(async (currency) => {
+      return await getCurrencyData(currency)
+    }),
+  )
+
+  logger.debug('currencyValues:', currencyValues)
+
+  const optionsData = await Promise.all(
+    cryptoCurrencies.map(async (currency, index) => {
+      return await getOptionsData(currency, currencyValues[index])
+    }),
+  )
+  // Return derivatives data mapped by currency
+  return optionsData.reduce((obj, data, idx) => {
+    obj[cryptoCurrencies[idx]] = data
+    return obj
+  }, {})
+}
+
+const getCurrencyData = async (currency) => {
+  const config = {
+    url: currencyEndpoint,
+    params: { currency },
+  }
+
+  const response = await Requester.request(config)
+  const path = ['result', currency]
+  return Requester.validateResultNumber(response.data, path)
+}
+
+const getOptionsData = async (currency, exchangeRate) => {
+  const config = {
+    url: bookDataEndpoint,
+    params: { currency, kind: 'option' },
+  }
+
+  try {
+    const response = await axios(config)
```

Comment:
Can we use `Requester.request()` here too?

Reply:
This api call pulls options data without granularity (deribit don't seem to support a filter). 
Since Requester.request logs the entire response data, it entails a significant amount of logging in this case.
The direct call to axios can be replaced with Requester.request() in case a "silent" call mode is added to the Requester, that would skip logging of the whole response

Reply:
In addition to the logging, the Requester will do a few extra things like retrying the request if the initial query failed. We have a few other calls we make that outputs unnecessary amounts of logging, so it's likely that the log level for this will be changed in Requester. I think it would be best if we stick with Requester.request() for now, even though it creates extra logging at the moment.

Reply:
updated to use Requester instead of the direct axios call

Reply:
Thanks. I just confirmed that we have a story to reduce the logging output of the Requester calls, so this should be taken care of.